src/app-exception-filters/main-exception.filter.ts

```diff
@@ -4,12 +4,13 @@ import {
   ArgumentsHost,
   HttpException,
   Logger,
+  LoggerService,
 } from '@nestjs/common';
 import { Response } from 'express';
 
 @Catch()
 export class MainExceptionFilter implements ExceptionFilter {
-  private readonly logger: Logger = new Logger();
+  private readonly logger: LoggerService = new Logger();
```

Comment:
tu można:
private readonly logger: Logger,

i podać Loggera do tablicy providers w module, gdzie następuje impleentacja tego filtra (tutaj w app.module)

Reply:
` private readonly logger = new Logger(MyService.name); `

czemu po prostu tak nie zrobicie? :D

Reply:
@mapawel to nie zadziała, testowałem. Z resztą Logger jest zaszyty w NestJs, więc można go użyć w każdym meijscu aplikacji bez wstrzykiwania w moduł
@Parodix007 poprawiłem :) bawiłem się trochę i zapomniałem to finalnie poprawić